code/drasil-code/lib/Language/Drasil/Code/Imperative/Modules.hs

```diff
@@ -378,9 +395,9 @@ constraintViolatedMsg q s c = do
     print v,
     printStr $ ", but is " ++ s ++ " to be "] ++ pc
 
--- | Generates statements to print descriptions of constraints, using words and
```

Comment:
Why did these comments get deleted?

Reply:
I am not sure, it may have happened during a merge conflict. I will add back these comments